core/src/main/scala/kafka/server/AddPartitionsToTxnManager.scala

```diff
@@ -63,7 +73,42 @@ class AddPartitionsToTxnManager(config: KafkaConfig, client: NetworkClient, time
   val verificationFailureRate = metricsGroup.newMeter(VerificationFailureRateMetricName, "failures", TimeUnit.SECONDS)
   val verificationTimeMs = metricsGroup.newHistogram(VerificationTimeMsMetricName)
 
-  def addTxnData(node: Node, transactionData: AddPartitionsToTxnTransaction, callback: AddPartitionsToTxnManager.AppendCallback): Unit = {
+  def addTxnData(
+    transactionalId: String,
+    producerId: Long,
+    producerEpoch: Short,
+    verifyOnly: Boolean,
```

Comment:
We will add this functionality later, but AddPartitionsToTxnManager currently only supports verifyOnly. In part 2, we will send AddPartitionsToTxnRequests that actually add the partition.

I suppose this is ok since replicaManager hard codes true for now.

Reply:
Ah. That's right. I think that it is better to remove it in this case. Unused params are always bad...